redash/query_runner/pinot.py

```diff
@@ -0,0 +1,135 @@
+try:
+    import pinotdb
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_DATETIME, TYPE_FLOAT, TYPE_STRING, TYPE_INTEGER, TYPE_BOOLEAN
+from redash.utils import json_dumps
+
+import requests
+from requests.auth import HTTPBasicAuth
+import logging
+
+logger = logging.getLogger(__name__)
+
+PINOT_TYPES_MAPPING = {
+    "BOOLEAN": TYPE_BOOLEAN,
+    "INT": TYPE_INTEGER,
+    "LONG": TYPE_INTEGER,
+    "FLOAT": TYPE_FLOAT,
+    "DOUBLE": TYPE_FLOAT,
+    "STRING": TYPE_STRING,
+    "BYTES": TYPE_STRING,
+    "JSON": TYPE_STRING,
+    "TIMESTAMP": TYPE_DATETIME,
+}
+
+
+class Pinot(BaseQueryRunner):
+    noop_query = "SELECT 1"
+    controller_uri = "http://localhost:9000"
+    username = None
+    password = None
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "brokerHost": {"type": "string", "default": "localhost"},
+                "brokerPort": {"type": "number", "default": 8099},
+                "brokerScheme": {"type": "string", "default": "http"},
+                "controllerURI": {"type": "string", "default": "http://localhost:9000"},
```

Comment:
```suggestion
                "controllerURI": {"type": "string", "default": ""},
```

This won't be the correct default for the majority of users. This default assumes that `controllerURI` runs _within the same Docker container as Redash_.